packages/styled-components/src/models/test/StyleSheetManager.test.js

```diff
@@ -202,51 +203,6 @@ describe('StyleSheetManager', () => {
     div.parentElement.removeChild(div);
   });
 
-  describe('ssr', () => {
-    it('should extract CSS outside the nested StyleSheetManager', () => {
```

Comment:
don't really understand what this test is trying to accomplish since refs don't work in an SSR use case

Reply:
yep, looks kind of useless. Might've tested CSS insertion into different targets, but it doesn't look like it does this correctly anyway